src/components/Breadcrumbs.js

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { css } from '@emotion/react';
+import { Link } from '@newrelic/gatsby-theme-newrelic';
+
+const Breadcrumbs = ({ segments, separator }) => {
+  return (
+    <div
+      css={css`
+        margin: 2em 0;
+        > :first-child {
+          margin-left: 0;
+        }
+        > * {
+          margin-left: 0.5em;
+        }
+      `}
+      aria-label="breadcrumb"
+    >
+      {segments
+        .map((segment) => {
+          const elem = segment.url ? (
+            <Link to={segment.url}>{segment.name}</Link>
+          ) : (
+            segment.name
+          );
+          return <span key={`breadcrumb-${segment.name}`}>{elem}</span>;
+        })
+        .reduce((prev, curr) => [
+          prev,
+          React.createElement('span', {}, separator),
```

Comment:
One option, if we wanted to avoid creating a new react component and HTML element, is to use emotion and the `:after` pseudo-element:

```jsx
segments.map((segment) => (
  <Link
    css={css`
      :after {
        content: "${separator}";
      }
    `}
    /* other props / JSX */
))
```

Reply:
ah yes, good ol' psuedo selector. didn't think of that. i like it!